extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.autoplay.js

```diff
@@ -2,23 +2,28 @@ define('wikia.articleVideo.featuredVideo.autoplay', [
 	'ext.wikia.adEngine.adContext',
 	'wikia.abTest',
 	'wikia.articleVideo.featuredVideo.cookies',
-], function (adContext, abTest, featuredVideoCookieService) {
+	require.optional('ext.wikia.adEngine.ml.rabbit')
+], function (adContext, abTest, featuredVideoCookieService, rabbit) {
+	'use strict';
+	var isAutoplayDisabledByRabbit = adContext.get('rabbits.ctpDesktop');
 	var inFeaturedVideoClickToPlayABTest = abTest.inGroup('FV_CLICK_TO_PLAY', 'CLICK_TO_PLAY');
 
-	function isAutoplayEnabled() {
-		return featuredVideoCookieService.getAutoplay() !== '0' &&
-			// adContext.isEnabled('wgArticleVideoAutoplayCountries') &&
-			!adContext.get('rabbits.ctpDesktop') &&
-			!inFeaturedVideoClickToPlayABTest;
-	}
-
-	function inNextVideoAutoplayEnabled() {
-		// return adContext.isEnabled('wgArticleVideoNextVideoAutoplayCountries');
-		return true;
-	}
-
 	return {
-		isAutoplayEnabled: isAutoplayEnabled,
-		inNextVideoAutoplayEnabled: inNextVideoAutoplayEnabled
-	}
+		isAutoplayDisabledByRabbits: function () {
+			var queenPrediction = rabbit.getPrediction('queendesktop');
```

Comment:
What if `rabbits === undefined`? We should check whether `rabbit` module is available here. Or remove `require.optional`